src/eko/couplings.py

```diff
@@ -62,6 +62,8 @@ def expanded_lo(ref, beta0, lmu):
 def expanded_nlo(ref, beta0, beta1, lmu):
     """Compute expanded solution at NLO.
 
+    From apfel
```

Comment:
maybe write that this is the default expression found everywhere, e.g. in |APFEL| eq. ...